### DISCUSSION TITLE
Using `getLastEventId ` in Laravel Lumen

### DISCUSSION THREAD

**Discussion by androidacy-user** [getsentry/sentry-laravel#555] [Q&A] [answered]
Hello!

We’d like to create an error page with a sentry feedback form for the couple routes that return HTML, if a request results in an error

However, app(‘sentry’) is not available in lumen. However, we were able to find a workaround for toTraceParent, so is there a more direct way we get that info?

Thanks!

**Comment by stayallive** [comment, marked as answer]
Hi @androidacybot,

You should not use `toTraceParent` this is not the last event ID but the current transaction ID. Unless you are using performance monitoring this value will be empty or invalid, it also includes other data next to the transaction ID, so it's not really meaningful in the feedback context.

The last event ID comes from the "hub" in the Sentry SDK and you can get the hub using: `\Sentry\SentrySdk::getCurrentHub()`, from there you can request the last event ID like so: `\Sentry\SentrySdk::getCurrentHub()->getLastEventId()`. This "hub" is what you would resolve from the container using `app('sentry')`.

Let me know if that worked!

**Reply by androidacy-user** [reply]
@stayallive the toTraceParent was irrelevant for that, it's just an example of how I worked around lumen not exposing some things Laravel would.

**Reply by androidacy-user** [reply]
Worked perfectly, thanks!
